config/rbac/role.yaml

```diff
@@ -5,6 +5,13 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
+    - ""
+  resources:
+    - secrets
+  verbs:
+    - list
+    - watch
```

Comment:
(nitpick): I think we are not watching any secret yet.

Reply:
If this is not in place I see an error in the manager logs:
``` 
E0427 09:02:56.827289       1 reflector.go:140] pkg/mod/k8s.io/client-go@v0.26.4/tools/cache/reflector.go:169: Failed to watch *v1.Secret: failed to list *v1.Secret: secrets is forbidden: User "system:serviceaccount:kyma-system:eventing-auth-manager-controller-manager" cannot list resource "secrets" in API group "" at the cluster scope
```

SO I'd suggest to keep it.